tests/test_app/test_io_new.py

```diff
@@ -445,3 +449,32 @@ def test_write_db_load_db(fasta_dir, tmp_dir):
         m = writer(orig, identifier=m.unique_id)
         read = reader(m)
         assert orig == read
+
+
+'''
+def test_findall():
```

Comment:
delete these, since `findall` is to be discontinued and we will be dropping support for zip's